R/activate.R

```diff
@@ -156,22 +156,27 @@ renv_activate_prompt <- function(action, library, prompt, project) {
   if (!ask)
     return(FALSE)
 
-  fmt <- lines(
-    "",
-    "This project has not yet been activated.",
-    "Activating this project will ensure the project library is used when %s() is called.",
-    "Please see `?renv::activate` for more details.",
-    ""
+  renv_activate_prompt_impl(project)
+
+
+}
+
+renv_activate_prompt_impl <- function(project = NULL) {
+  choices <- c(
```

Comment:
We might also want some way to cache the user's response per-session so we don't bug them multiple times (maybe a separate PR)